README.md

```diff
@@ -2,60 +2,81 @@
 
 Version 0.1
 
-Overview of the input/output requirements for VECTOR.
+Overview of the input/output requirements for VECTOR. The typical frontend
+workflow will be (1) a call to the `/geometry` endpoint to get a unique `id`,
+followed by (2) a call to the `/singlepoint/:id` endpoint with the desired
+input parameters to compute all of the output values. Finally, (3) a call to the
+`/image/:id` endpoint will return an image of the satellite geometry that was
+used in the computations.
 
 ## Endpoints
 
-- /api/image : POST
+- /geometry : POST
 
-This endpoint accepts a VRML `.wrl` file for upload and returns a `.png` image of the file.
+    This endpoint is for uploading files and accepts a VRML `.wrl` file.
+    It will return a unique `id` to be associated with the upload.
 
-- /api/singlepoint : POST
+- /geometry/sat_name : POST
 
-This endpoint is to submit single-point calculation parameters to.
-The backend will then parse the payload and run VECTOR with the proper inputs to get the return values.
-The return from the VECTOR code will then be put in the response payload to send back to the frontend.
+    This endpoint says we want the requested satellite's geometry file that we
+    already have locally, so there is no need to upload a file. This endpoint
+    will return a unique `id` to be associated with the requested satellite.
+
+    The named satellites available are:
+  - `sorce`, The SORCE satellite
+  - `cubesat1u`, A 1U CubeSat
+  - `deployable3u`, A 3U CubeSat with solar panels
```

Comment:
totally knitpicky, but maybe we don't want the articles capitalized after the comma?

Reply:
Good to catch those things, because they wouldn't get addressed later I'm sure :) Thanks!